src/locales/ko/dialogue.ts

```diff
@@ -1134,26 +1134,26 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "lorelei": {
     "encounter": {
-      1: `No one can best me when it comes to icy Pokémon! Freezing moves are powerful!
-                $Your Pokémon will be at my mercy when they are frozen solid! Hahaha! Are you ready?`,
+      1: `얼음 포켓몬은 나를 대적할 사람이 없지! 빙결의 길은 강해!
```

Comment:
```suggestion
      1: `얼음포켓몬을 내보내면 대적할 사람이 없지! 상대를 얼린다는 건 매우 강력한 공격이야.
```
"move"의 의미가 명확히 안 사는 것 같아서 [레츠고 대사 원문](https://youtu.be/Cy39z1HhLXo?si=rE9sudltMltrYclI&t=136)을 그대로 쓰는 걸 제안해 봅니다.

Reply:
이게 나은 것같아요 ㅋㅋ어제 렛츠고피카츄 칸나 대면 장면 고민하다가 그렇게 썼는데 확인감사합니다